docs/en/modules/bme680.md

```diff
@@ -0,0 +1,159 @@
+# BME680 module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2017-10-28 | [vsky279](https://github.com/vsky279) | [vsky279](https://github.com/vsky279) | [bme680.c](../../../app/modules/bme680.c)|
+
+Copyright (C) 2017 - 2018 Bosch Sensortec GmbH
```

Comment:
What is this copyright for?

Reply:
I'm using parts of the [bme680.c](https://github.com/BoschSensortec/BME680_driver/blob/2a51b9c0c1899f28e561e6701caa22cb23201cfc/bme680.c) driver from Bosch - it says we should retain the copyright. I think maybe just one line should be enough.

Reply:
I saw that, yes. However, I thought leaving the copyright in the source code would be enough. It feels somewhat out of place here and makes it look like Bosch contributed the NodeMCU module (and wrote this documentation).

Reply:
This seems reasonable. I will make a block in the code with these functions and will put the (c) note there.